api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/PurchasesDebugViewAPI.kt

```diff
@@ -0,0 +1,14 @@
+package com.revenuecat.apitester.kotlin
+
+import androidx.compose.runtime.Composable
+import com.revenuecat.purchases.debugview.DebugRevenueCatBottomSheet
+import com.revenuecat.purchases.debugview.DebugRevenueCatScreen
+
+@Suppress("unused")
+private class PurchasesDebugViewAPI {
+    @Composable
```

Comment:
Haha this is basically just like `@ViewBuilder`.

Reply:
Yup it's super similar to SwiftUI :P